jaas/src/main/java/com/griddynamics/jagger/jaas/storage/JaggerTestDbConfigDao.java

```diff
@@ -0,0 +1,19 @@
+package com.griddynamics.jagger.jaas.storage;
+
+import com.griddynamics.jagger.jaas.storage.model.JaggerTestDbConfig;
+
+import java.util.List;
+
+/**
+ * DAO contract for {@link com.griddynamics.jagger.jaas.storage.model.JaggerTestDbConfig}.
+ */
+public interface JaggerTestDbConfigDao {
```

Comment:
I am going to rename it to just DbConfigEntity. Because it is an abstract one and there is nothing specific to Jagger or Jagger Test Db to reflect in its name.